lib/phoenix_token_auth/mailer.ex

```diff
@@ -25,6 +25,7 @@ defmodule PhoenixTokenAuth.Mailer do
   Both config fields have to be functions returning binaries. welcome_email_subject receives the user and
   welcome_email_body the user and confirmation token.
   """
+  def send_welcome_email(user = %{email: email}, confirmation_token) when email == nil do end
```

Comment:
This is my last comment, after that I will merge. Promise!
This line throws a warning, because confirmation_token isn't used. I think we can just remove the whole line anyway, right?